docs/docs/data-structures/tree/index.mdx

```diff
@@ -20,467 +18,45 @@ A `SharedTree` has the following characteristics:
--   Removal of items never overrides inserting (or moving in) items. For example, if one client removes items 10-15, and another client simultaneously inserts an item at index 12, the original items 10-15 are removed, but new item is inserted between item 9 and the item that used to be at index 16. This is true regardless of the order of the remove and insert operations.
-
-For the meaning of "simultaneously", see [Types of distributed data structures](../overview.mdx).
-
-```typescript
-removeAt(index: number)
-```
-
-Removes the item at the given `index`.
-
-```typescript
-removeRange(start?: number, end?: number)
-```
-
-Removes the items indicated by the `start` index (inclusive) and `end` index (exclusive). If the end index is omitted, every item from the start index to the end of the array is removed. If the start index is omitted, it defaults to 0. So, calling `removeRange()` removes all the items in the array.
-
-###### Move methods
+To get started working with `SharedTree` in your application, read this [quick start guide](../../start/tree-start.mdx).
+The major programming tasks for using `SharedTree`s that are described in this doc are:
 
-Array nodes have three methods that move items within an array or from one array node to another. When moving from one array node to another, these methods must be called from the destination array node. Note that in all of the following, the `T` can be any type that is derived from an object that is returned by a call of `SchemaFactory.array()`, such as the `Notes` and `Items` classes in the sticky notes example.
+-   Defining a schema for the `SharedTree`. As you build out from prototype to full production application, you can add to this schema and create additional schemas for additional `ShareTree`s.
+-   Initializing the `TreeView` object.
+-   Creating code that reads and edits the nodes of the `SharedTree`.
 
-```typescript
-moveToStart(sourceStartIndex: number, sourceEndIndex: number, source?: T)
-```
+### Usage
 
-Moves the specified items to the start of the array. Specify a `source` array if it is different from the destination array.
+Provided below are links to more detailed information about `SharedTree` usage and its available APIs.
 
-```typescript
-moveToEnd(sourceStartIndex: number, sourceEndIndex: number, source?: T)
-```
+- [Node Types](./node-types.mdx): outlines the types that are stored by a `SharedTree`
+	- [Schema Definition](./schema-definition.mdx): how the structure of a `SharedTree` is defined
+	- [Reading and Editing](./reading-and-editing.mdx): how a `SharedTree` is read and edited through the APIs provided on the different node types
 
-Moves the specified items to the end of the array. Specify a `source` array if it is different from the destination array.
+## Other
 
-```typescript
-moveToIndex(index: number, sourceStartIndex: number, sourceEndIndex: number, source?: T)
-```
+### TreeView Object
```

Comment:
If this page is going to be a TOC, then we should move the rest of the content in an appropriate sub-page.